source/process/additional-asset-guidelines.rst

```diff
@@ -0,0 +1,32 @@
+===================================
+Additional Asset Guidelines
+===================================
+
+Guidelines for Screenshot Development of Mattermost User Interface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The purpose is to highlight and illustrate the best features of Mattermost user interface.
+
+Instructions:
+
+1. Please take the screenshot from the Mattermost Desktop app (it has a nicer border).
+
+2. Left-hand side: Include Favorite, Public, and Private channels on the left-hand side. Make sure that you scroll all the way to the top in the channel list on the left-hand side for the screenshot.
+
+3. Favorites channel list: Include one Public channel, a Direct Message channel with someone who is online, as well as a Group Direct Message channel using people with short names to avoid truncation.
+
+4. Center pane: Make the center pane one of the Favorited channels. Make sure that the heading of the center pane is fully visible to avoid truncation. Also, make the first message in the center pane fully visible right below the channel header (otherwise it will look messy).
+
+5. Screen size: Provide screenshots that are 20-30% larger than needed - this will make it easier for the screenshot to be included in banners. Close the right-hand sidebar and shrink your screen horizontally to make it narrower. However, leave the screen wide enough until the "mobile send button" no longer appears in the bottom-right corner. Make sure not to cut off any sides of the page in the screenshot.
+
+6. Profiles: Profile pictures and real people’s names should be filled out throughout the screenshot. For the profile picture, use either normal-looking Avatar heads or stock photography.
+
+7. Clarity: Do not include an "unread posts" indicator, but you can have a few mentions. Also, avoid showing any text with acronyms and abbreviations. Do not show any bugs or defects.
+
+8. One easy way to create demo screenshots is to have a secondary pre-release account. Use [first.last]+blah@mattermost.com to create a second account. This will make triggering notifications for yourself easier (for the screenshot). You can also use a second browser to login to the second account while taking the screenshot.
+
+
+Guidelines for Animated GIFs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We should not create animated GIFs with two images that look very similar, since visually it will look like a mistake.
```

Comment:
propose active voice (since visually it looks like..)

Also, should we use the word `we` in docs?